modules/core/src/main/scala/org/tessellation/config/types.scala

```diff
@@ -41,8 +53,72 @@ object types {
   case class SnapshotConfig(
     heightInterval: NonNegLong,
     globalSnapshotPath: Path,
-    fallbackTriggerTimeout: FiniteDuration,
+    timeTriggerInterval: FiniteDuration,
     inMemoryCapacity: NonNegLong
   )
 
+  case class SoftStakingConfig(
+    address: Address,
+    nodes: NonNegInt,
+    startingOrdinal: SnapshotOrdinal,
+    softNodesPercentage: Percentage,
+    testnetNodes: NonNegInt,
+    testnetAddress: Address
+  )
+
+  case class DTMConfig(
+    address: Address,
+    monthly: Amount
+  )
+
+  case class StardustConfig(
+    address: Address,
+    percentage: Percentage
+  )
+
+  case class RewardsConfig(
+    epochs: PosInt,
+    epochDurationInYears: PosDouble,
+    totalPool: Amount,
+    baseEpochReward: Amount,
+    softStaking: SoftStakingConfig,
+    dtm: DTMConfig,
+    stardust: StardustConfig,
+    rewardsPerEpoch: NonEmptyMap[EpochProgress, Amount]
+  )
+
+  object RewardsConfig {
+
+    val default = RewardsConfig(
+      epochs = 4,
+      epochDurationInYears = 2.5,
+      totalPool = Amount(160000000000000000L),
+      baseEpochReward = Amount(85333333320000000L),
```

Comment:
Unused config values. The `totalPool` can be hardcoded in the `RewardsSuite` since it will be the only place using it.